src/rofl/common/crofconn.cc

```diff
@@ -1691,17 +1704,21 @@ unsigned int crofconn::segment_group_desc_stats_reply(
       msg->set_stats_flags(msg->get_stats_flags() |
                            rofl::openflow13::OFPMPF_REPLY_MORE);
     }
-    rofsock.send_message(msg);
+    /* when enforcing queueing, there are only two return values possible:
+     * MSQ_QUEUED and MSG_QUEUED_CONGESTION. We return the result received for
+     * the last fragment. */
+    msg_result = rofsock.send_message(msg, /*enforce-queueing*/true);
```

Comment:
enforce-queueing (for all the segment functions) is essentially a bugfix, label and changelog should be updated accordingly